contracts/forwarding/IForwarderPayable.sol

```diff
@@ -0,0 +1,14 @@
+/*
+ * SPDX-License-Identifier:    MIT
+ */
+
+pragma solidity ^0.4.24;
+
+
+interface IForwarderPayable {
```

Comment:
We should likely inherit from the fee interface here. Otherwise, tooling wouldn't know how much (and which token) you should pay to forward.

Reply:
I completely agree, the reason why I didn't do that is to avoid the following compiler error:
```
TypeError: Overriding function changes state mutability from "nonpayable" to "payable".
```

Reply:
Hmm weird, [`IForwarderFee`](https://github.com/aragon/aragonOS/blob/next/contracts/common/IForwarderFee.sol#L8) shouldn't have any problem with conflicting interface declarations though, right?

Reply:
Sorry I misread, I thought you were saying about inheriting from `IForwarder`. Sure, let's give it a try